backend/README.md

```diff
@@ -1,12 +1,38 @@
 # Flotilla backend
 
+The backend of flotilla is created using ASP.NET.  
+Useful documentation of concepts and features in the .NET frameworks can be found 
+[here](https://docs.microsoft.com/en-us/dotnet/fundamentals/).
+
+## Setup
+To set up the backend on Windows/Mac, install visual studio and include the "ASP.NET and web development" workload during install.  
+If you already have visual studio installed, you can open the "Visual Studio Installer" and modify your install to add the workload.
+
+To set up the backend on linux..  
+CHRISTIAN PLEASE HELP ME  
```

Comment:
To set up the backend, install .NET for linux [here](https://docs.microsoft.com/en-us/dotnet/core/install/linux).